Content.Shared/Interaction/SharedInteractionSystem.cs

```diff
@@ -850,46 +851,14 @@ public void ThrownInteraction(EntityUid user, EntityUid thrown)
         }
         #endregion
 
-        #region Drop
-        /// <summary>
-        /// Activates the Dropped behavior of an object
-        /// Verifies that the user is capable of doing the drop interaction first
-        /// </summary>
-        public bool TryDroppedInteraction(EntityUid user, EntityUid item)
-        {
-            if (!_actionBlockerSystem.CanDrop(user)) return false;
-
-            DroppedInteraction(user, item);
-            return true;
-        }
-
-        /// <summary>
-        ///     Calls Dropped on all components that implement the IDropped interface
-        ///     on an entity that has been dropped.
-        /// </summary>
         public void DroppedInteraction(EntityUid user, EntityUid item)
         {
-            var dropMsg = new DroppedEvent(user, item);
+            var dropMsg = new DroppedEvent(user);
             RaiseLocalEvent(item, dropMsg);
             if (dropMsg.Handled)
-            {
                 _adminLogSystem.Add(LogType.Drop, LogImpact.Low, $"{ToPrettyString(user):user} dropped {ToPrettyString(item):entity}");
-                return;
-            }
-
-            Transform(item).LocalRotation = Angle.Zero;
```

Comment:
why was this removed?

Reply:
After RobustToolbox/pull/2576, entities in containers should already have zero local rotation.

Though now that I think about it RobustToolbox/pull/2594 might screw with that, seeing as it makes entities preserve world rotation when changing parents, so I guess I should add it back in.